templates/_components/_lister.twig

```diff
@@ -0,0 +1,109 @@
+{# _components/_lister #}
+
+{#
+  Lister
+  Generate a list of entry views
+  
+  @param  entries          {array}    List of Elements
+  @param  kind             {string}   Name of view type that you want to use for each list item eg: 'teaser' | 'title' etc.
+  @param  class            {string}   Class names for list wrapper element
+  @param  list_class       {string}   Class names for <ul> tag element
+  @param  list_attributes  {array}    Attributes for <ul> tag element
+  @param  item_class       {string}   Class names for <li> tag element
+
+
+
+  Configuration
+  =============
+  There is a configurable option available which help save
+  some repetition when using the component for multiple use cases.
+
+  ListerProfiles
+  ----------
+  A set of pre-configured values for list generation. Define the desired
+  configurations in the `listerProfiles` array using the following options:
+  
+  @option  view  {string}  Name of view type that you want to use for each list item eg: 'teaser' | 'title' etc.
+  @option  list  {string}  Class names for <ul> tag element
+  @option  item  {string}  Class names for <li> tag element
+
```

Comment:
A `wrap` option is also supported which will add a class to a wrapper around the list. (See line 88)

Reply:
Added to comment section.

Reply:
But which section did you add it to? Please read the code and figure out _where_ this parameter is supported and hence where should it be documented.

Reply:
Sorry, moved the comments to the right section.